src/test/fuzz/script.cpp

```diff
@@ -133,15 +109,11 @@ FUZZ_TARGET_INIT(script, initialize_script)
     (void)ScriptToAsmStr(script, true);
 
     UniValue o1(UniValue::VOBJ);
-    ScriptPubKeyToUniv(script, o1, true, true);
-    ScriptPubKeyToUniv(script, o1, true, false);
+    ScriptPubKeyToUniv(script, o1, true);
```

Comment:
b0770cee942e17f6ed3d4ded wherever this function is called
```suggestion
    ScriptPubKeyToUniv(script, /* out */ o1, /* include_hex */ true);
```